osc_sdk.c

```diff
@@ -5074,7 +5074,7 @@ int osc_load_region_from_conf(const char *profile, char **region)
 }
 
 static int osc_load_cert_from_conf_(const char *profile, char **cert, char **key,
-	char **proxy)
+				    char **proxy, char **endpoint)
```

Comment:
Can you remove blank space please?

Reply:
well, they where baldy indent before, and this in fact fix it :)